web/public/locales/no-NB.json

```diff
@@ -1,270 +1,301 @@
-    }
+  "aggregateButtons": {
+    "country": "land",
+    "zone": "prisområde"
   },
-  "country-panel": {
-    "source": "kilde",
-    "carbonintensity": "CO₂-intensitet",
-    "lowcarbon": "Lavt CO₂-utslipp",
-    "renewable": "Fornybar",
-    "electricity": "Strøm",
-    "electricityproduction": "Strømproduksjon",
-    "electricityconsumption": "Strømforbruk",
-    "bysource": "etter kilde",
-    "emissions": "CO₂-utslipp",
-    "helpfrom": "med hjelp fra",
-    "addeditsource": "<a href=\"%s\" target=\"_blank\">Tilføy eller rediger kilde</a>",
-    "noDataAtTimestamp": "Data midlertidig utilgjengelig for tidspunktet",
-    "noLiveData": "Sanntidsdata er midlertidig utilgjengelig for dette området",
-    "dataIsDelayed": "Data fra denne regionen kan være %s timer forsinka. Sanntidsdata kan derfor ikke vises.",
-    "dataIsEstimated": "Data for dette området er forsinket. Nåtidsverdiene er estimater. <a href=\"https://github.com/electricityMaps/electricitymaps-contrib/wiki/Estimation-methods\" target=\"_blank\">Les mer om estimeringsmetodene våre</a>.",
-    "estimatedDisclaimer": "Sanntidsdataene er estimert basert på datakilden samt modellforutsetninger. Ingen garantier for at dataene er nøyaktig.",
-    "helpUs": "Hjelp oss å finne årsaken til problemet ved å undersøke <a href=\"%s\" target=\"_blank\">loggene</a> eller ta kontakt med datatilbyder.",
-    "noParserInfo": "Vi har ennå ingen datakilder om området.<br/>Vil du hjelpe med å løse det? Du kan bidra ved å <a href=\"%s\" target=\"_blank\">tilføye regionale data</a>.",
-    "now": "Nå",
-    "disabledPriceReasons": {
-      "licensing": "Elektrisitetspriser er utilgjengelige for dette området på grunn av rettighetsbegrensninger"
-    },
-    "estimated": "anslag",
-    "aggregated": "aggregert",
-    "exchangesAreMissing": "Utveksling er tatt hensyn til i kalkulasjonene en vises for tiden ikke. Les mer om våre historiske aggregeringer <a href=\"https://github.com/electricityMaps/electricitymaps-contrib/wiki/Historical-aggregates\" target=\"_blank\">her</a>.",
-    "estimatedTooltip": "Sanntidsdata for denne el-markedssonen er estimert ved hjelp fra kilden basert på antagelser i modellen.",
-    "aggregatedTooltip": "Data for denne el-markedssonen er aggregert ut fra historiske verdier for hver time",
-    "aggregationDisabled": "Data er ikke tilgjengelig for det valgte tidspunktet på grunn av manglende detaljnivå på tidsserie. Velg en lengere tidsperiode.",
-    "by-source": {
-      "emissions": "Karbonutslipp etter kilde",
-      "electricity-production": "Elektrisitetsproduksjon etter kilde",
-      "electricity-consumption": "Elektrisitetsforbruk etter kilde",
-      "total-emissions": "Totale karbonutslipp etter kilde",
-      "total-electricity-production": "Total elektrisitetsproduksjon etter kilde",
-      "total-electricity-consumption": "Total elektrisitetsforbruk etter kilde"
+  "aria": {
+    "label": {
+      "changeTheme": "Endre utseende",
```

Comment:
Should this not be more like "Endre tema" than "Endre utseende"? As we just allow them to change theme and not individual looks?